src/unix.rs

```diff
@@ -109,3 +77,7 @@ pub fn enable_raw_mode() -> Result<TermMode, Error> {
     set_term_mode(&term)?;
     Ok(orig_term)
 }
+
+pub fn stdin() -> std::io::Result<std::io::Stdin> { Ok(std::io::stdin()) }
+
+pub fn path(filename: &String) -> std::path::PathBuf { std::path::PathBuf::from(filename) }
```

Comment:
Not sure why CI is not giving a warning, but when I run clippy locally I get this:

```
error: writing `&String` instead of `&str` involves a new object where a slice will do
  --> src/unix.rs:83:23
   |
83 | pub fn path(filename: &String) -> std::path::PathBuf { std::path::PathBuf::from(filename) }
   |                       ^^^^^^^ help: change this to: `&str`
   |
   = note: `-D clippy::ptr-arg` implied by `-D warnings`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg
```

Can you use `&str` instead?

Reply:
sure - done